guides/common/modules/proc_importing-all-available-ansible-playbooks.adoc

```diff
@@ -0,0 +1,13 @@
+
+[id="importing-all-available-ansible-playbooks_{context}"]
+= Importing All Available Ansible Playbooks
+
+You can import all the available Ansible playbooks from collections installed on the {SmartProxy}.
+
+.Procedure
+. On the target host, import the Ansible playbooks:
```

Comment:
```suggestion
. Import the Ansible playbooks using the following API request:
```
The API call does not have to be made from a host and it does not relate to a particular host. (API calls can be technicaly made from a local machine or anywhere that can access the SatServer over HTTPS.)